tests/unit/test-extension-runners/test.firefox-android.js

```diff
@@ -128,6 +128,7 @@ function prepareSelectedDeviceAndAPKParams(
     startFirefoxAPK: sinon.spy(() => Promise.resolve()),
     setupForward: sinon.spy(() => Promise.resolve()),
     clearArtifactsDir: sinon.spy(() => Promise.resolve()),
+    checkOrCleanArtifacts: sinon.spy(() => Promise.resolve()),
```

Comment:
In this test file we should actually add a test case to assert that this spy is being called as expected (when we expect it to be called, and with the parameters we expect) to actually cover the changes applied to the firefox-android test runner (and also make sure to properly cover the possible scenarios, e.g. when it should have been called to only check for the old artifacts and when it is supposed to be called to remove those old artifacts).